client/extensions/woocommerce/app/products/product-form.js

```diff
@@ -108,3 +113,24 @@ export default class ProductForm extends Component {
 		);
 	}
 }
+
+function mapApiToProps( apiClient, ownProps, state ) {
+	const { siteId, productId } = ownProps;
+	const apiProduct = apiClient.selectors.getProduct( productId, { freshness: 30 * SECOND } );
```

Comment:
I read the FULL pr description more carefully and answered my question.

Reply:
> Freshness is awesome, and makes sense in many cases, but I fear if we have a page where we query multiple endpoints ( thinking the dashboard for example ) this could get pretty chatty.

Definitely. In future PRs, I plan to add a feature that will stop fetching all data when the user is inactive. I'm also open to other suggestions.

>  it is also something I could see unsuspecting developers ( cough me ) using incorrectly and creating a bunch of API traffic.

I'm not sure how to police our own developers on this, or if we should. There are plenty of use cases where fairly frequent polling might make sense. Maybe a minimum freshness that can be enforced as a part of the API settings? (in state/wc-api for this case) Or maybe we could set a minimum freshness setting on each endpoint, so if there's an endpoint that should be polled more or less often, we could set that.

> The product description did not refresh on the edit page. Guessing there is some tinymce funkyness going on there.

I saw the same behavior. I think our TinyMCE editor component is probably dropping the ball here.